### DISCUSSION TITLE
Should we normalize text in transcriptions?

### DISCUSSION THREAD

**Discussion by willfant** [coqui-ai/TTS#1385] [General Q&A] [answered]
For a good dataset transcription, should we normalize the text? I mean, I notice that for instance the ljspeech metadata has 2 versions of the text. Ex: audio1|i'm 50 years old.|i am fifty years old.

It is that a requirement? Not sure if the others datasets have the same rule. If it's a requirement is there a way do to that in multilingual dataset without doing it manually?

Thanks!

**Comment by erogol** [comment, marked as answer]
You should normalize the text, either before training or writing your own text cleaner here 
https://github.com/coqui-ai/TTS/blob/main/TTS/tts/utils/text/cleaners.py
